src/feature/common/DarkModeToggle.tsx

```diff
@@ -19,10 +20,7 @@ export const DarkModeToggle = () => {
     <Button
       size="icon"
       variant="ghost"
-      onClick={() => {
```

Comment:
don't we still need this to not break all of the mantine stuff?

Reply:
We put the maintine stuff inside toggleTheme function to make it not break

Reply:
Oh oops i didn't see that

Reply:
yeah just so that onClick is only taking one single prop. If this is bad practice or might cause issues later, I'll revert back the changes.